### PR TITLE
Rename freebsd.c to bsd.c. Add NetBSD support.

### PR DESCRIPTION
**Pull request by jsonn**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -68,6 +68,18 @@ CTFMERGE = /usr/bin/true
...
+MANSECT = 1
+endif
+
+ifeq ($(UNAME_S),NetBSD)
+CC != if [ -x /usr/bin/clang ]; then echo /usr/bin/clang; else echo /usr/bin/gcc; fi
+
+CTFMERGE != if [ -x /usr/bin/ctfmerge ]; then echo /usr/bin/ctfmerge; else echo /usr/bin/true; fi
+CTFCONVERT != if [ -x /usr/bin/ctfconvert ]; then echo /usr/bin/ctfconvert; else echo /usr/bin/true; fi
+
 CFILES += plat/freebsd.c plat/unix_common.c
```
Comment (jclulow) [inline Makefile]: Should this line also be `plat/bsd.c` now, too?

File: plat/bsd.c [hunk 2]
```diff
@@ -25,7 +25,11 @@
 #include "dynstr.h"
 #include "plat/unix_common.h"
 
+#ifdef __NetBSD__
+#define	SERIAL_DEVICE	"/dev/tty01"
```
Comment (jclulow) [inline plat/bsd.c]: Is `/dev/tty01` always "the second serial port (COM2) on the machine"?

File: Makefile (hunk 1 above)
Comment (jsonn) [inline Makefile]: Bah, merge error when copying from the test machine.

File: plat/bsd.c (hunk 2 above)
Comment (jsonn) [inline plat/bsd.c]: Unless either /dev or kernel have been explicitly configured otherwise, yes.

**Comment by travispaul** [comment]
Hi @jclulow,

The merge error in CFILES has been corrected, and tested on the latest NetBSD and SmartOS releases.  Is there anything I can do to help get this change merged? Is any testing needed on other platforms?

Thanks!

**Comment by rmustacc** [comment]
Resolved in  4edaed6. Thanks all.

**Pull request closed without merging**